apps/mobile-wallet/src/features/tokenDisplay/tokenDetailsModal/TokenDetailsModal.tsx

```diff
@@ -50,6 +54,55 @@ const TokenDetailsModal = withModal<TokenDetailsModalProps>(({ id, tokenId, addr
   )
 })
 
+interface TokenAnimatedBackgroundProps {
+  tokenId: Token['id']
+  addressHash?: AddressHash
+}
+
+const TokenRoundedCard = ({ tokenId, addressHash }: TokenAnimatedBackgroundProps) => {
+  const theme = useTheme()
+  const [dominantColor, setDominantColor] = useState<string>()
+  const token = useAppSelector((state) => selectFungibleTokenById(state, tokenId))
```

Comment:
You can extract the logoURI here to simplify the code and improve performance

```suggestion
  const tokenLogoUri = useAppSelector((state) => selectFungibleTokenById(state, tokenId)?.logoURI)
```

Reply:
Done

Reply:
In which commit? Can't find the change to cherry pick it in the rebased branch

Reply:
Feel free to deprecate this PR in favor of:
- https://github.com/alephium/alephium-frontend/pull/1134

Feel free to push anything you want there.